docker-app/qfieldcloud/filestorage/management/commands/migratefilestorage.py

```diff
@@ -84,7 +90,8 @@ def handle(self, *args, **options) -> None:
 
             exit(1)
 
-        if len(projects) == 0:
+        nb_projects = len(projects)
```

Comment:
```suggestion
        projects_count = len(projects)
```

If you don't mind, most of the codebase does the numbering with `_count` suffix.